src/app/contextBroker/contextBroker.cpp

```diff
@@ -52,7 +52,7 @@
 * To override the security added with the certificate, curl can always be called using the
 * CLI option '--insecure'.
 */
-#include <stdio.h>
+#include <stdio.h>                              // snprintf       
```

Comment:
```suggestion
#include <stdio.h>                              // snprintf
```

To avoid extra trailing whitespace.